amazon-kinesis-client/src/test/java/software/amazon/kinesis/retrieval/fanout/FanOutConsumerRegistrationTest.java

```diff
@@ -156,9 +156,9 @@ public void testRegisterStreamConsumerThrottled() throws Exception {
             consumerRegistration.getOrCreateStreamConsumerArn();
         } finally {
             verify(client, times(RSC_RETRIES))
-                    .registerStreamConsumer(eq(createRegisterStreamConsumerRequest()));
+                    .registerStreamConsumer(any(RegisterStreamConsumerRequest.class));
             // Verify that DescribeStreamConsumer was called for at least RegisterStreamConsumer retries + 1 at start.
-            verify(client).describeStreamConsumer(eq(createDescribeStreamConsumerRequest(null)));
+            verify(client).describeStreamConsumer(any(DescribeStreamConsumerRequest.class));
```

Comment:
"Verify that DescribeStreamConsumer was called for at least RegisterStreamConsumer retries + 1 at start."

This is not what we are verifying. This verifies that we are calling it exactly once, I'm not saying this is guaranteed to be an implementation detail that we are testing, but take a cursory glance to see if this is a worthwhile verification.